client/locations-list-map.js

```diff
@@ -382,8 +382,15 @@ function createRequesterMarkerContent(entry, separator) {
     website,
   } = entry;
 
+  const header = separator ? ce('h5', 'separator', ctn(name)) : ce('h5', null, ctn(name));
+  const headerPartnerLink = createPartnerLinkIcon(entry.row_id);
+
+  if (headerPartnerLink) {
+    ac(header, headerPartnerLink);
+  }
+
   // Text to go into InfoWindow
-  const contentTags = separator ? [ce('h5', 'separator', ctn(name))] : [ce('h5', null, ctn(name))];
+  const contentTags = [header];
```

Comment:
we have a bit of overlap in this area, but it'll be easy to resolve and I can do it on my branch.

Reply:
The links are the "D&E" images that match the site favicon.

Reply:
Yeah, it's not that I don't see the links, it's that they're not there. And the `body` tag doesn't have the partner data attributes set...

Reply:
Do you mean that you don't see the red-circled icon/link at the test URL shown?

![image](https://user-images.githubusercontent.com/21227171/83798564-70f7ec00-a672-11ea-9357-9ec5753b8eed.png)

Reply:
Yes, I think that is what you mean. I went to a different machine and don't see them either.

Reply:
Yes, I mean exactly that.

I also mean (when I investigated the issue) that the variables your code is setting in the router aren't being set in my browser. I'm using Brave, it's a Chromium based browser.

<img width="948" alt="2020-06-04_11-51-02" src="https://user-images.githubusercontent.com/1222063/83798867-beb42a80-a659-11ea-87e3-0ebe2d2896f8.png">

<img width="423" alt="2020-06-04_11-51-14" src="https://user-images.githubusercontent.com/1222063/83798875-c07dee00-a659-11ea-8d40-940dd2e72a02.png">

Reply:
I also don't see them in Firefox. They're there in Safari.

Reply:
Hmm, this must be some kind of caching issue, but I can't get it to resolve in Brave...

I just opened Chrome (a browser I don't use on this computer) and it worked. So I guess don't worry about it, some sort of local quirk.